archive-editorial.php

```diff
@@ -4,16 +4,16 @@
 
 <!-- main content -->
 
-<main id="main-content">
+<main id="main-content" class="container">
```

Comment:
no logic was retroactively fixing other templates and didn't realise the container was in the wrong place